services/trainer-hours-service/src/main/java/crm/trainerhoursservice/model/WorkloadSummary.java

```diff
@@ -0,0 +1,35 @@
+package crm.trainerhoursservice.model;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.index.Indexed;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Document(collection = "workload_summary")
+@Builder
+@ToString
+@Getter
+@Setter
+public class WorkloadSummary {
+    @Id
+    private String id;
+    @Indexed
```

Comment:
Trainer First Name & Trainer Last Name should be part of the index (https://stackoverflow.com/questions/14040942/how-to-use-spring-data-mongo-compoundindex-with-sub-collections)